app/src/forms/email/emailService.js

```diff
@@ -287,8 +287,11 @@ const service = {
   submissionReceived: async (formId, submissionId, body, referer) => {
     try {
       const { configData, contexts } = await buildEmailTemplate(formId, submissionId, EmailTypes.SUBMISSION_RECEIVED, referer, { body });
-
-      return service._sendEmailTemplate(configData, contexts);
+      if (contexts[0].to.length) {
+        return service._sendEmailTemplate(configData, contexts);
+      } else {
+        return {};
+      }
```

Comment:
Really don't think this is the right place to be doing this? It's not the email service deciding based on the CHES contexts whether to send, it should be the CONTROLLER deciding to invoke email service based on the request
See other comment below:

I'm not sure this should be based on checking contexts in the email service. Aren't the contexts set by the supplied email address in the body, so shouldn't it check a level or 2 higher to even see if email service should be called at all?

So check body email address at the controller level. The code flow is really "do I need to do anything about an email here?" when controlling the submission POST

IE, controller.createSubmission line 126